cms/djangoapps/contentstore/views/course.py

```diff
@@ -458,12 +458,14 @@ def filter_ccx(course_access):
     return courses_list.values(), in_process_course_actions
 
 
-def _accessible_libraries_iter(user):
+def _accessible_libraries_iter(user, org=None):
     """
-    List all libraries available to the logged in user by iterating through all libraries
+    List all libraries available to the logged in user by iterating through all libraries.
+
+    If 'org' is present, only libraries from that org will be returned.
     """
     # No need to worry about ErrorDescriptors - split's get_libraries() never returns them.
-    return (lib for lib in modulestore().get_libraries() if has_studio_read_access(user, lib.location.library_key))
+    return (lib for lib in modulestore().get_libraries(org=org) if has_studio_read_access(user, lib.location.library_key))
```

Comment:
@efischer19 Unfortunately it turns out that this way of filtering is case-sensitive. In contrast, the way I limit courses is not case sensitive, and this inconsistency might be confusing to users (it was to me!). I wonder if there is any way to specify a case-insensitive org match.